packages/components/src/Card/Card.mdx

```diff
@@ -40,6 +40,12 @@ import { Card } from "@jobber/components/Card";
   </Card>
 </Playground>
 
+## Design & Usage Guidelines
+
+The primary utility of a card is its' grouping of elements by proximity within a clear boundary, clearly marked by a header.
```

Comment:
"grouping of elements by proximity within a clear boundary" sounds really technical and I don't think would help me make design decisions. What's the actual goal of a card? To group similar tasks so our users can scan and prioritize information more easily, or something like that?

Reply:
I tweaked the component description and this part, I think now it better gets at what I'm trying to say?